server/views/partials/waitingListStatusDescription.njk

```diff
@@ -0,0 +1,13 @@
+{% macro waitingListStatusDescription(waitingListStatus) -%}
+  {% if waitingListStatus == 'PENDING' %}
+    Pending
+  {% elseif waitingListStatus == 'APPROVED' %}
+    Approved and on the waitlist
+  {% elseif waitingListStatus == 'DECLINED' %}
+    Rejected
+  {% elseif waitingListStatus == 'ALLOCATED' %}
```

Comment:
Could use an enum here instead of comparing to strings?

Reply:
yes, better idea. Will look at it

Reply:
updated. Also moved the file under activities